packages/nx/src/command-line/release/config/filter-release-groups.spec.ts

```diff
@@ -473,12 +526,7 @@ describe('filterReleaseGroups()', () => {
           projects: ['lib-a'],
           changelog: false,
           docker: undefined,
-          version: {
-            conventionalCommits: false,
-            generator: '',
-            generatorOptions: {},
-            groupPreVersionCommand: '',
-          },
+          version: createVersionConfig(),
```

Comment:
I'm not a fan of this abstraction, every test previously was standalone and self-documenting, I don't think this is adding any value but has that cost